scholia/app/templates/base.html

```diff
@@ -86,6 +86,18 @@
 	 }
 	 catch(e) { }
 
+	 console.log(item.claims.P6782[0])
+	 try {
+	     var rorID = item.claims.P6782[0].mainsnak.datavalue.value;
+	     detailsList.push( '<img alt="ROR icon" src="' +
+			       'https://pbs.twimg.com/profile_images/1086592528038707200/qVDdvqOH_400x400.jpg' +
+			       '" width="16" height="16" hspace="4" /> <a href="https://ror.org/' +
+			       encodeURI( rorID ) + '">' + rorID + '</a> ');
+	 }
+	 catch(e) {
+	     console.log("ERROR: " + e)
```

Comment:
Would this report an error if the external ID is not there?

Reply:
Yes, I guess it would. You did not want that, right? I'll remove it.

Reply:
Removed